definitions/install.yaml

```diff
@@ -483,12 +483,87 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
-  name: policyviolation
+  name: kyvernoRole
 rules:
-- apiGroups: ["kyverno.io"]
+# Dynamic creation of webhooks and events 
+- apiGroups:
+  - '*'
   resources:
+  - events
+  - mutatingwebhookconfigurations
+  - validatingwebhookconfigurations
+  - certificatesigningrequests
+  - certificatesigningrequests/approval
+  verbs:
+  - create
+  - delete
+  - get 
+  - list
+  - patch
+  - update
+  - watch
+# get the roleRef for incoming api-request user
+- apiGroups:
+  - '*'
+  resources:
+  - rolebindings
+  - clusterrolebindings
+  - configmaps
+  verbs:
+  - watch
```

Comment:
Need `get` and `list` as well?

Reply:
@realshuting 
Currently, we do not use `get` & `list` verbs on resources [`rolebindings`, `clusterrolebindings `, `configmaps `], hence not specified.

Reply:
Can we retrieve the userInfo from an admission request without these operations?

Reply:
Didn't understand, Which section of code are your referring to?

Reply:
Here it gets the list of roles and cluster roles for the incoming api-request:
https://github.com/nirmata/kyverno/blob/cf59326c64f973d715134e2123584f6d62252f3f/pkg/userinfo/roleRef.go#L21-L22

Reply:
The `List` operation above is on the informer cache(which is built using the watch on the resource). Here we do not query api-server for List operation.